tools/js-sdk-release-tools/src/hlc/generateMgmt.ts

```diff
@@ -63,7 +63,7 @@ export async function generateMgmt(options: {
         let outputPackageInfo = getOutputPackageInfo(options.runningEnvironment, options.readmeMd, undefined);
 
         try {
-            logger.logGreen(`Installing dependencies for ${changedPackageDirectory}...`);
+            logger.info(`Installing dependencies for ${changedPackageDirectory}...`);
```

Comment:
It would be better to have a same prefix when starting a new step, such as 'start to ...'. With this, the message can be filtered out and print in the CI check result easily.